src/pages/Calculator/Steps/SpeciesSelection/PlantList.js

```diff
@@ -90,14 +106,14 @@ const PlantList = ({
               sx={{
                 color: "#DA7059",
                 position: "absolute",
-                top: "2rem",
+                top: "1rem",
                 fontStyle: "italic",
                 fontWeight: "bold",
                 bgcolor: "primary.light",
                 opacity: "80%",
               }}
             >
-              Not Recommended for Planting Date
+              Recommend Planting Date: {startDate} - {endDate}
```

Comment:
can you change the text to `Not recommended for planting before ${startDate}` or `Not recommended for planting after ${endDate}` depending on if it's before or after the reliable establishment period?